### ISSUE TITLE
[dev.icinga.com #4091] Ensure issue existence before starting feature

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#121]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/4091

**Created by mhein on 2013-05-07 07:43:42 +00:00**

Assignee: _(none)_
Status: _Resolved_  (closed on _2013-06-07 07:50:03 +00:00_)
Target Version: _1.0-2_
Last Update: _2013-06-12 07:36:26 +00:00 (in Redmine)_

---

As a developer I need to add a new issue before I am able to start a new feature branch.

---

**Parent Task:** [#4089](https://dev.icinga.com/issues/4089)
